Bash_scripts/HW15/README.md

```diff
@@ -0,0 +1,10 @@
+Dockerfile syntax:
+1.	Difference between ARG and ENV
+2.	What is VOLUME? Create an example of a Dockerfile using VOLUME
+3.	Create NOBODY user and run container as this user
```

Comment:
User should be created inside your Dockerfile

Reply:
![hw15p3_change](https://github.com/Engelko/DevOps_course/blob/HW15/Bash_scripts/HW15/hw15p3_change.png)


Was that meant in the assignment?

Reply:
This is correct